src/main/java/com/rst/bootcamp/controller/BootCampController.java

```diff
@@ -0,0 +1,80 @@
+package com.rst.bootcamp.controller;
+
+import java.net.URI;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
```

Comment:
Unused import
should clean code before PR is raised

Reply:
Done